CHANGELOG.md

```diff
@@ -1,13 +1,18 @@
 # Changelog
 All notable changes to this project will be documented in this file.
+=======
```

Comment:
this line is unnecessary, it turns the line above into a large header

Reply:
Fixed